CONTRIBUTING.rst

```diff
@@ -36,5 +37,128 @@ Pull Request. Contributions may include:
 * Helping to onboard new contributors
 * Other related activities
 
+Development
+===========
 
-Check `README.rst <README.rst>`_ development section instructions.
\ No newline at end of file
+Requirements
+-------------
+
+- Python >=3.9
+- Pipenv
+
+Getting the source code
+-----------------------
+
+`Fork <https://docs.github.com/en/get-started/quickstart/fork-a-repo>`_ the
+repository on `GitHub <https://github.com/vmware/repository-service-tuf-cli>`_ and
+`clone <https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository>`_
+it to your local machine:
+
+.. code-block:: console
+
+    git clone git@github.com:YOUR-USERNAME/repository-service-tuf-cli.git
+
+Add a `remote
+<https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/configuring-a-remote-for-a-fork>`_ and
+regularly `sync <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork>`_ to make sure
+you stay up-to-date with our repository:
+
+.. code-block:: console
+
+    git remote add upstream https://github.com/vmware/repository-service-tuf-cli
+    git checkout main
+    git fetch upstream
+    git merge upstream/main
+
+Preparing the environment
+-------------------------
+
+After installing Python, install the pipenv tool:
+
+.. code:: shell
+
+    $ pip install pipenv
+
+
+Create a virtual environment for this project:
+
+.. code:: shell
+
+    $ pipenv shell
+
+
+Install the requirements from the Pipfile.
+
+The flag -d will install the development requirements:
+
+.. code:: shell
+
+    $ pipenv install -d
+
+
+.. note::
+
+    macOS running on MacBook M1
+
+    For developers, after the above command, run:
+
+    .. code:: shell
+
+        $ pip uninstall cryptography cffi -y
+        $ pip cache purge
+        $ LDFLAGS=-L$(brew --prefix libffi)/lib CFLAGS=-I$(brew --prefix libffi)/include pip install cffi cryptography
+
+
+Running checks with pre-commit:
+
+The pre-commit tool is installed as part of the development requirements.
+
+To automatically run checks before you commit your changes you should run:
+
+.. code:: shell
+
+    $ make precommit
+
+This will install the git hook scripts for the first time, it will update to the
+latest versions of the hooks and run the pre-commit tool.
```

Comment:
```suggestion
This will install the git hook scripts for the first time, update to the
latest versions of the hooks and run the pre-commit tool.
```